packages/core/src/config.ts

```diff
@@ -42,7 +42,7 @@ export interface MedplumSourceInfraConfig {
   maxAzs: ValueOrExternalSecret<number>;
   rdsInstances: ValueOrExternalSecret<number>;
   rdsInstanceType: ValueOrExternalSecret<string>;
-  rdsInstanceVersion: ValueOrExternalSecret<string>;
+  rdsInstanceVersion?: ValueOrExternalSecret<string>;
```

Comment:
It seems like this is intended to be optional, this was causing a type error in the tests since it was marked as required, though in the `MedplumInfraConfig` it is treated as optional

Reply:
Nice, good catch